C/services/common/notification_manager.cpp

```diff
@@ -797,7 +797,29 @@ string NotificationManager::getJSONRules()
 		RulePlugin* builtinRule = this->findBuiltinRule((*it).first);
 		if (builtinRule)
 		{
-			ret += this->getPluginInfo(builtinRule->getInfo());
+			// Get builtin pluginInfo
+			string bRule = this->getPluginInfo(builtinRule->getInfo());
+
+			// TODO: remove this and apply substitution in getPluginInfo()
```

Comment:
Please fix this TODO

Reply:
@monika-Dianomic  We can fix it after Fledge branch merge

This Notification server change can be merged now

Another PR will that care of new plugin option in Fledge